_posts/2021-10-11-021.md

```diff
@@ -1,27 +1,28 @@
 ---
 layout: post
-title: '#021 git remote add nome url'
-image: "https://res.cloudinary.com/jesstemporal/image/upload/f_auto/v1642878674/gitfichas/pt/021/thumbnail_l7xjqu.jpg"
+pretitle:
+title: Adicionando
+subtitle: um remote
+command: git remote add nome url
+descriptors:
+  - command: comando para gerenciar remotes
+  - part1: criar novo remote
+  - part2: nome do remote, por exemplo origin
+  - part3: caminho para o remote
+author: "@jtemporal"
+number: "021"
+mermaid: true
+use_static_image: true
```

Comment:
Since you are using this, can you generate the SVG with the command below and add the image? Another option is to remove this line. Thanks :)

```
python3 scripts/generate_images_only.py "021.md"
```